tools/make/kube.mk

```diff
@@ -82,6 +82,7 @@ kube-install-image: image.build $(tools/kind) ## Install the EG image to a kind
 run-conformance: ## Run Gateway API conformance.
 	kubectl wait --timeout=5m -n gateway-system deployment/gateway-api-admission-server --for=condition=Available
 	kubectl wait --timeout=5m -n envoy-gateway-system deployment/envoy-gateway --for=condition=Available
+	kubectl wait --timeout=5m -n gateway-system pod -l job-name=gateway-api-admission --for=jsonpath='{.status.phase}'=Succeeded
```

Comment:
Any specific reason for using `jsonpath` over `--for=condition=complete`?

```
$ kubectl get job/gateway-api-admission -n gateway-system -o yaml
...
status:
  completionTime: "2022-11-28T19:27:06Z"
  conditions:
  - lastProbeTime: "2022-11-28T19:27:06Z"
    lastTransitionTime: "2022-11-28T19:27:06Z"
    status: "True"
    type: Complete
  ready: 0
  startTime: "2022-11-28T19:26:57Z"
  succeeded: 1
```

`Succeeded` specifies the number of pods that reached phase "Succeeded". If the number of pods is increased for the Job and one fails, the wait will succeed.

From [Kube API docs](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#jobstatus-v1-batch):
> When a Job is completed, one of the conditions will have the type "Complete" and status true.

Reply:
updated